crates/ruff_db/src/vendored.rs

```diff
@@ -1,15 +1,18 @@
 use std::borrow::Cow;
 use std::collections::BTreeMap;
 use std::fmt::{self, Debug};
-use std::io::{self, Read};
-use std::sync::{Mutex, MutexGuard};
+use std::io::{self, Cursor, Read};
```

Comment:
nit: there's now some places in this file where we refer to `std::io::Cursor` as `io::Cursor`, and other places in this file where we just refer to it as `Cursor`. It would be nice to be consistent inside this module.